static/build/.tmp_update/script/update_networking.sh

```diff
@@ -48,7 +49,21 @@ else
 fi
 }
 
+sync_time() {
+    if [ -f "$sysdir/config/.NTPState" ]; then
+        while true; do
+            if [ ! -f "/tmp/ntp_run_once" ]; then
+                break
+            fi
 
+            if ping -q -c 1 google.com > /dev/null 2>&1 ; then
+                ntpdate time.google.com &
```

Comment:
Backgrounding this, and then just sleeping 2 secs seems non-deterministic ? Why not just wait for ntpdate to finish, then break ?